backend/helpers/PayPalWebhookHelper.py

```
@@ -0,0 +1,495 @@
+# ---------- helpers to extract useful bits from the event ----------
+
+def _get_event_type(evt: Dict[str, Any]) -> str:
+    return (evt or {}).get("event_type") or ""
+
+def _res(evt: Dict[str, Any]) -> Dict[str, Any]:
+    return (evt or {}).get("resource") or {}
+
+def _safe_float(v: Any) -> float:
+    try:
+        return float(v)
+    except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)